docs/platform/howto/disk-autoscaler.md

```diff
@@ -57,20 +57,33 @@ the available disk storage capacity and depends on a service type:
 -   Aiven for Apache Cassandra®: 35% of the available disk storage
     capacity
 -   Aiven for OpenSearch®: 75% of the available disk storage capacity
--   All the other Aiven service types: 85% of the available disk storage
-    capacity
+-   All the other Aiven service types: 85% of the available disk storage capacity
+
 :::
 
+## Limits and limitations
+
+- Maximum storage capacity that Aiven Autoscaler can allocate for your service is
+  limited by:
+  - Maximum disk capacity set for the
+    [Aiven Autoscaler endpoint](#change-max-disk-space)
+  - Maximum [DDS](/docs/platform/howto/add-storage-space) storage capacity supported on
+    your service plan
+- When triggered, the autoscaling process takes a moment. Meanwhile, the service disk
+  might get full, and your service might enter the read-only mode. In such cases, the
+  service is back to normal as soon as the autoscaling process completes unless Aiven
+  Autoscaler's disk capacity limits are exceeded.
+- Disk autoscaling works on fully running services only and cannot happen during
```

Comment:
We might be able to drop this limitation, worth checking with @ilpon

Reply:
It requires some work (to ignore old nodes) and I'd like to first see that DDS works when applied manually during maintenance operations before making autoscaler do it.

In other words, I'd leave it for future enhancement.